core/server/scripts/build.sh

```diff
@@ -12,6 +12,8 @@ source "${script_dirpath}/_constants.env"
 
 BUILD_DIRNAME="build"
 
+DEFAULT_SKIP_DOCKER_IMAGE_BUILDING=false
```

Comment:
super nit: how do you compare this to DEFAULT_DOCKER_BUILD_IMAGE = false; and then you pass in true above

Reply:
So, those are different, on docker_image_builder we have `push_to_registry_container`, and on build.sh this is `skip_docker_image_building`. So, slightly different. I could switch `skip_docker_image_building` to `build_docker_image` and turn the default to true, no strong opinion here 🤷